packages/api/src/middleware/auth.ts

```diff
@@ -118,10 +119,13 @@ function authenticator(): RequestHandler {
       throw new ForbiddenError(`user is suspended`);
     }
 
-    req.user = user;
+    // make copies in case someone mutates these
+    req.token = { ...tokenObject };
```

Comment:
This is causing several tests to fail with `access disallowed to api keys`

Reply:
Hmmm

Reply:
Understood it! problem was when we didnt have an API key here, we were still setting the `token` var to an empty object. Fixed, good catch.